tests/test_georaster.py

```diff
@@ -544,33 +544,83 @@ def test_reproject(self, example: str) -> None:
         # Reference raster to be used
         r = gr.Raster(example)
 
-        # Create an artificial raster with different bounds, shape and resolution
-        new_data = np.ones(r.shape)
-        transform = list(r.transform)
-        new_transform = rio.transform.Affine(
-            transform[0] / 2,
-            transform[1],
-            (r.bounds.left + r.bounds.right) / 2,
-            transform[3] / 2,
-            transform[4],
-            (r.bounds.bottom + r.bounds.top) / 2,
+        # -- Check proper errors are raised if nodata are not set -- #
```

Comment:
Is the "--" to make the comment a bit of a "header" a PEP8 convention?

Reply:
I don't think it is a convention. I used to have ## or ### like in markdown, but flake8 or black don't like it.
I never found anything about writing some sort of headers in comments, but I do like to split my code in larger blocks. I think I've seen this elsewhere and found it was a good idea.